tsdb/db.go

```diff
@@ -1104,10 +1104,11 @@ func BeyondSizeRetention(db *DB, blocks []*Block) (deletable map[ulid.ULID]struc
 	deletable = make(map[ulid.ULID]struct{})
 
 	walSize, _ := db.Head().wal.Size()
+	checkpointsSize := wal.GetCheckpointSize(db.Head().wal.Dir())
```

Comment:
Let us log this/these errors, it will be helpful in debugging if someone's size based retention is not working.

Reply:
And also the error in line 1107